pandas/core/frame.py

```diff
@@ -6837,6 +6837,31 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
         Returns
         -------
         DataFrame
+        
+        Examples
+        --------
+        >>> df = pd.DataFrame(
+        ...  {"name" : ['Rahul','Arijit','Tushar','Amit']}, 
+        ...    index = [
+        ...        [70, 67, 95, 89], 
+        ...        ['CSE', 'EEE', 'IT', 'ECE'],
+        ...        ['B,'C','A+','A']
+        ...    ]
+        ... )
+        >>> df
+                     name
+        70 CSE B    Rahul
+        67 EEE C   Arijit
+        95 IT  A+  Tushar
+        89 ECE A     Amit
+        
+        >>> df.reorder_levels([0,2,1])
+                     name
+        70 B  CSE   Rahul
+        67 C  EEE  Arijit
+        95 A+ IT   Tushar
+        89 A  ECE    Amit
```

Comment:
The best examples are those which are as simple and minimal as possible indicating solely the effect of the method.

```suggestion
        >>> df = pd.DataFrame({"name": ["Rahul", "Amit"]}, 
        ...         index=[[1, 2], ["A", "B"], ["Red", "Blue"]]) 
        >>> df
                      name
        1 A Red      Rahul
        2 B Blue      Amit
       
        >>> df.reorder_levels([0,2,1])
                      name
        1 Red A      Rahul
        2 Blue B      Amit
```